pkg/storaged/client.js

```diff
@@ -593,15 +572,15 @@ function init_model(callback) {
         }
     }
 
-    pull_time().then(function() {
-        read_os_release().then(function () {
-            enable_features().then(function() {
-                query_fsys_info().then(function(fsys_info) {
+    pull_time().then(() => {
+        read_os_release().then(os_release => {
+            client.os_release = os_release;
+
+            enable_features().then(() => {
+                query_fsys_info().then((fsys_info) => {
                     client.fsys_info = fsys_info;
 
-                    client.storaged_client.addEventListener('notify', function () {
-                        client.update();
-                    });
+                    client.storaged_client.addEventListener('notify', client.update);
```

Comment:
This is not a valid transformation in general. `client.update` is now called with different arguments and a different `this`. It will work in this specific case, but it might break in the future and if applied elsewhere.  Let's not do this.  (Also, it has nothing to do with the topic of this PR, no?)

Reply:
No, it's unrelated, I just didn't want to  mix arrows and old-style full-function calls, so took the opportunity. I put back the extra function wrapper now, it's not important.